src/RecyclableMemoryStreamManager.cs

```diff
@@ -707,9 +698,9 @@ internal void ReportStreamOverCapacity(Guid id, string tag, long requestedCapaci
             this.StreamOverCapacity?.Invoke(this, new StreamOverCapacityEventArgs(id, tag, requestedCapacity, this.MaximumStreamCapacity, allocationStack));
         }
 
-        internal void ReportUsageReport(long smallPoolInUseBytes, long smallPoolFreeBytes, long largePoolInUseBytes, long largePoolFreeBytes)
+        internal void ReportUsageReport()
         {
-            this.UsageReport?.Invoke(this, new UsageReportEventArgs(smallPoolInUseBytes, smallPoolFreeBytes, largePoolInUseBytes, largePoolFreeBytes));
+            this.UsageReport?.Invoke(this, new UsageReportEventArgs(this.smallPoolInUseSize, this.smallPoolFreeSize, this.LargePoolInUseSize, this.LargePoolFreeSize));
```

Comment:
worth handling exceptions? usage reporting is not the primary purpose of the manager

Reply:
Exception handling could impact the performance of this method by making it non-inlinable, so I'd rather not without some stronger evidence that it should be done here rather than in the user's code.